### DISCUSSION TITLE
Question for results in DPA2 paper

### DISCUSSION THREAD

**Discussion by duduRUHappy** [deepmodeling/deepmd-kit#3594] [Q&A]
Regarding Table 2 in the DPA2 paper: Zero-shot generalization errors, I understand that for all DownStream tasks, DPA2 MT should be a model trained on the entire pre-training dataset. Therefore, for the same downstream task, the results of DPA2 MT should be consistent. However, at the highlighted positions in the figure below, both lines are for AlMgCu-D, but the RMSE for DPA2 MT in terms of Energy and Force are different, with Energy showing a larger deviation between the two. 
![image](https://github.com/deepmodeling/deepmd-kit/assets/49438740/b92092a1-ca91-4825-b8d5-6da7c2ac9722)
[4.DPA2 Towards a universal large atomic model for molecular and material simulation.pdf](https://github.com/deepmodeling/deepmd-kit/files/14740797/4.DPA2.Towards.a.universal.large.atomic.model.for.molecular.and.material.simulation.pdf)

**Comment by wanghan-iapcm** [comment]
The first zero-shot error was tested with the AlMgCu head while the second was tested with the Alloy head.

**Reply by duduRUHappy** [reply]
That makes sense now, thank you!
